backend/src/pong/game_invite/game-invite.spec.ts

```diff
@@ -6,12 +6,15 @@ import {
 import { ConfigModule } from '@nestjs/config';
 import { Test, TestingModule } from '@nestjs/testing';
 import { TypeOrmModule } from '@nestjs/typeorm';
+import { User } from '../../user/entities/user.entity';
```

Comment:
general archtitecture question, you added a game invite column in the users, and I'm not sure if we need those, since both players seem to be saved in the players array?

Reply:
Yea I'm not sure about this either, I think this will change when we start building the business logic of the game and chat, I think the entire Invite table might be removed in the end.

So the players are not saved in the players array, only a relationship is established as mentioned above.